Riot/Modules/Contacts/Views/FindYourContactsFooterView.swift

```diff
@@ -0,0 +1,81 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import Reusable
+
+@objc protocol FindYourContactsFooterViewDelegate {
+    func didTapFindYourContactsButton()
+}
+
+@objcMembers
+class FindYourContactsFooterView: UIView, NibLoadable, Themable {
+    
+    // MARK: - Properties
+    
+    weak var delegate: FindYourContactsFooterViewDelegate?
+    
+    @IBOutlet weak var containerView: UIView!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var messageLabel: UILabel!
+    @IBOutlet weak var button: CustomRoundedButton!
+    @IBOutlet weak var footerLabel: UILabel!
```

Comment:
Let's make all of these private. Except `button` I guess, although I would hide the outlet behind a public `actionEnabled" boolean or similar.